keyboards/gizmo_engineering/gk6/gk6.c

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2021 Gizmo Engineering
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "gk6.h"
+
+const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
```

Comment:
```suggestion
#ifdef RGB_MATRIX_ENABLE
const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
```

Reply:
Good call

Reply:
Also, again, will error out, but will be "fixed" when develop is merged in. 
```suggestion
const is31_led __flash g_is31_leds[DRIVER_LED_TOTAL] = {
```

Reply:
The `__flash` part is needed still.